eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/http/demo/AsyncPublishInstance.java

```diff
@@ -32,6 +32,9 @@
 
     public static Logger logger = LoggerFactory.getLogger(AsyncPublishInstance.class);
 
+    private final static RandomStringGenerator randomGenerator = new RandomStringGenerator.Builder()
```

Comment:
It seems RandomStringGenerator is thread safe, so we can just hold one instance in our application.
Can we add a new utils or factory to Wrapper RandomStringGenerator.
```java
public class InnerStringUtils {

	/**
	 *  generate random number string
	 * @param length string length
	 * @return
	 */
	public static String generateRandomNumber(int length) {
		return RandomStringGeneratorHolder.instance.generate(length);
	}

	public static class RandomStringGeneratorHolder() {
		private static RandomStringGenerator instance = new RandomStringGenerator.Builder()
				.withinRange('0', '9').build();
	}

}
```

Reply:
yes, a util is better